packages/gluestick-cli/package.json

```diff
@@ -40,6 +40,7 @@
     "fs-extra": "^2.0.0",
     "gluestick-generators": "^1.0.0-rc.8",
     "glob": "7.1.1",
-    "mkdirp": "0.5.1"
+    "mkdirp": "0.5.1",
+    "shelljs": "0.7.6"
```

Comment:
Do we need a whole new dependency `shelljs` just for using `which`? Is there any other way without adding this dep? If not, I am ok with it.